Documentation/ContentElements/EditingContent/Index.rst

```diff
@@ -1,30 +1,157 @@
-.. include:: /Includes.rst.txt
-
-
-.. _content-editing:
+..  include:: /Includes.rst.txt
+..  _content-editing:
 
 ===============
 Editing content
 ===============
 
-..  youtube:: JR3sOeRImTo
-
-------------
+..  image:: /Images/ManualScreenshots/ContentElements/EditContent.png
+    :alt: Editing content elements in the TYPO3 backend
+    :class: img-thumbnail float-end ms-1
+    :width: 250
 
 On a page, click the pencil icon for the content element you want to edit,
 then edit the text and make other changes as required.
 
+Click the edit button to start editing the content element
+
+..  contents:: Table of contents:
+
+..  _content-editing-change-text:
+
+Changing text and setting in a content element
+==============================================
+
+Once you clicked on the edit button you can change text, images and all settings
+that you have permissions for. There might be additional settings that only an
+advanced editor or an administrator can see and edit. Those might be hidden to
+you.
+
+..  figure:: /Images/ManualScreenshots/ContentElements/Edit.png
+    :alt: Content element edit form screenshot, demonstrating changed content which is highlighted
+
+    Changed content is highlighted. The save button is found on the top
```

Comment:
This text does not describe what happens on that image

Reply:
True, I put the wrong image, changed it, thanks for catching!